docs/core/tutorials/unloadability-howto.md

```diff
@@ -0,0 +1,362 @@
+# Using and debugging unloadability in .NET Core
+An ability to load and later unload a set of assemblies is one of the features that was missing until now in .NET Core. In .NET Framework, custom app domains are used for this purpose. But since .NET Core supports only a single default app domain, how can we achieve that goal in .NET Core?
```

Comment:
These first few paragraphs read a lot like a blog post to me. Also, we need to think of topics as future proof. So saying things like now, etc. should be avoided.

A suggestion on how to get this started:

```suggestion
Starting with .NET Core 3.0, the ability to load and later unload a set of assemblies is supported. In .NET Framework, custom app domains were used for this purpose, but .NET Core only supports a single default app domain.
```